docs/en/user_guides/4_train_test.md

```diff
@@ -1,110 +1,89 @@
-    checkpoint=dict(type='CheckpointHook', by_epoch=False, interval=4000),
-    sampler_seed=dict(type='DistSamplerSeedHook'))
-]
-```
-
-### Use pre-trained model
-
-Users can load a pre-trained model by setting the `load_from` field of the config to the model's path or link.
-The users might need to download the model weights before training to avoid the download time during training.
-
-```python
-# use the pre-trained model for the whole PSPNet
-load_from = 'https://download.openmmlab.com/mmsegmentation/v0.5/pspnet/pspnet_r50-d8_512x1024_40k_cityscapes/pspnet_r50-d8_512x1024_40k_cityscapes_20200605_003338-2966598c.pth'  # model path can be found in model zoo
-```
+## Training and testing on a single machine with a single GPU
 
 ### Training on a single GPU
 
 We provide `tools/train.py` to launch training jobs on a single GPU.
 The basic usage is as follows.
 
 ```shell
-python tools/train.py \
-    ${CONFIG_FILE} \
-    [optional arguments]
+python tools/train.py  ${CONFIG_FILE} [optional arguments]
 ```
 
 This tool accepts several optional arguments, including:
 
 - `--work-dir ${WORK_DIR}`: Override the working directory.
 - `--amp`: Use auto mixed precision training.
-- `--resume ${CHECKPOINT_FILE}`: Resume from a previous checkpoint file. If not specify, try to auto resume from the latest checkpoint in the work directory.
-- `--cfg-options ${OVERRIDE_CONFIGS}`: Override some settings in the used config, the key-value pair in xxx=yyy format will be merged into config file.
+- `--resume ${CHECKPOINT_FILE}`: Resume from a previous checkpoint file. If not specified, try to auto resume from the latest checkpoint in the work directory.
+- `--cfg-options ${OVERRIDE_CONFIGS}`: Override some settings in the used config, and the key-value pair in xxx=yyy format will be merged into the config file.
   For example, '--cfg-option model.encoder.in_channels=6'. Please see this [guide](./1_config.md#Modify-config-through-script-arguments) for more details.
-  Below is the optional arguments for multi-gpu test:
+
+Below are the optional arguments for the multi-gpu test:
+
 - `--launcher`: Items for distributed job initialization launcher. Allowed choices are `none`, `pytorch`, `slurm`, `mpi`. Especially, if set to none, it will test in a non-distributed mode.
 - `--local_rank`: ID for local rank. If not specified, it will be set to 0.
 
-**Note**:
-Difference between `--resume` and `load-from`:
-`--resume` loads both the model weights and optimizer status, and the iteration is also inherited from the specified checkpoint.
+**Note:** Difference between the argument `--resume` and the field `load-from` in the config file:
+`--resume` loads both the model weights and optimizer status and the iteration is also inherited from the specified checkpoint.
 It is usually used for resuming the training process that is interrupted accidentally.
 
 `load-from` only loads the model weights and the training iteration starts from 0. It is usually used for fine-tuning.
```

Comment:
The `Note` might not be required as `resume` doesn't support loading the specific checkpoint, and there might be confusion between `resume` and `load_from`.